app/backend/extract_tops.py

```
@@ -14,13 +14,36 @@
+    """Unload model from Ollama memory via native API."""
+    try:
+        response = requests.post(
+            f"{_OLLAMA_NATIVE_URL}/api/generate",
+            json={"model": model, "keep_alive": 0},
+            timeout=30,
+        )
+        if response.status_code == 200:
+            logger.info(f"Model '{model}' unloaded from Ollama memory")
+        else:
+            logger.warning(f"Unexpected response when unloading model: {response.status_code}")
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)